nhost/hasura-backend-plus
# refactor(auth): multiple refactoring, bug fixes and improvements

- rename change email/password routes, files and templates
- reshape typings, in particular regarding GraphQL requests
- simplify JWT verification in setting a getClaims function
- simplify getting account by email/id/ticket
- fix missing `new_email` column in the `auth.accounts` schema
- test the email change workflow
- add a `JWT_CLAIMS_NAMESPACE` option (see #163)
- rename `AUTH_MICROSOFT_ENABLE` to `WINDOWS_LIVE_ENABLE`
- change codecov workflow in GH test action
- switches back from argon2 to bcrypt (closes #176)
